tracer/test/test-applications/integrations/Samples.SpanLinks/Samples.SpanLinks.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+</Project>
```

Comment:
We probably don't need to add this project to git, since you already have tests for adding span links and validation serialization. These `Sample.*` are usually reserved for tests that require automatic instrumentation, since we need to run a separate process with the required environment variables.

Once we start using span link in auto-instrumentation, those will get validated through other means, like snapshot tests.

Reply:
That's my bad 😄 
@chojomok Let me know if you want me to remove it from being tracked from `git` or feel free to do so yourself

I just figured it'd be an easy entry point for `SampleHelpers` for reflection to get access to `AddSpanLink` since it isn't exposed and test if it the actual agent could deserialize the message pack.

But yeah I'd agree that it's probably best to remove it.

Reply:
git noob here - this is a case where I would add it to gitignore right? So future commits don't track it. 
And I would also have to remove it from the origin branch

Reply:
You can just delete the folder and push a commit with those deletes. If you want to keep the project to use locally, move it somewhere else outside of the git repo.

Reply:
So there are a few things you could do I think the simplest would be:

- Move the project folder (`Samples.SpanLinks`) outside of the `dd-trace-dotnet` directory
- Undo the `SampleHelper` changes
- Save the `.sln` which should update that file
- Commit that

Reply:
I ended up deleting the project folder trying to do it myself brainfarting and forgetting that I can just move the files. 
Should be fine though I don't think I need it anymore and can always try to set it up myself by referencing the history here if I do. 

Do we need to move the AddSpanLink out of SampleHelper? I feel like it could be useful later no?

Reply:
I think it's easy enough to add back in at a later point _if we need it_. 

Lucas pointed out that we'd probably just have an integration create span links and then we can just test it that way (calling `Span.AddSpanLink` directly) and won't need to use reflection

Reply:
bet - i removed it

Reply:
Thanks!